Exec/science/subch_planar/GNUmakefile

```diff
@@ -0,0 +1,31 @@
+PRECISION        = DOUBLE
+PROFILE          = FALSE
+DEBUG            = FALSE
+DIM              = 2
+
+COMP	         = gnu
+
+USE_MPI          = TRUE
+USE_OMP          = FALSE
+
+USE_GRAV         = TRUE
+USE_REACT        = TRUE
+
+USE_SHOCK_VAR = TRUE
+
+USE_MODEL_PARSER = TRUE
+
```

Comment:
we should add

```
USE_SIMPLIFIED_SDC = TRUE
```

here as well, since that's how we always intend for this to be run